app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -30,6 +34,14 @@ class AuthRepository @Inject constructor(
 		firebaseAuth.signOut()
 	}
 
+	suspend fun userProfile() = withContext(ioDispatcher) {
+		Profile(
+			photo = firebaseAuth.currentUser?.photoUrl,
```

Comment:
Do we expect, for this particular situation, that the currentUser can be null here ? If not, we can just force not null like for deleting account :) Is there a way to enter this screen not being logged in ? 

Also we can use currentUser only once, using one of kotlin Standard extensions functions to not repeat ourselves in calling it :) 

I am also thinking about the fields, which can really be null ? i am not sure that email can be null for example, when the user is using email ?

Reply:
I changed it and I follow the approach with currentUser will be not null and only Profile parameters could be nullable and I used `let` extension scope function :P